server/src/service/PkiService.js

```diff
@@ -298,6 +298,39 @@ exports.splitChainIntermediateCertificate = async (body) => {
   return chainInfo;
 };
 
+/**
+ *
+ */
+exports.retrieveFirstAndRemainingIntermediateChainCerts = async (intermediateChain) => {
+  const beginCertRegex = /(?=-----BEGIN)/g;
+
+  let chainInfo = [];
+  let remainingIntermediateChainInfo = '';
+  let count = ((intermediateChain && intermediateChain.match(/BEGIN/g)) || []).length;
```

Comment:
Shoouldn't this line use the beginCertRegex var?

Reply:
I created this function using the one you shared with me for reference: "splitChainIntermediateCertificate" and it uses the same logic.

Reply:
I'm trying to understand why there are three similar regexes being used:
`const beginCertRegex = /(?=-----BEGIN)/g;`
`match(/BEGIN/g)`
`match(/-----BEGIN CERTIFICATE-----/g`

I'd like to use just one

Reply:
There are similar definitions in other functions/methods, let's use just one `const` with the correct regex